sentry-android-core/src/main/java/io/sentry/android/core/LifecycleWatcher.java

```diff
@@ -147,4 +159,10 @@ AtomicBoolean isRunningSession() {
   TimerTask getTimerTask() {
     return timerTask;
```

Comment:
We lock access to this but give away the ref here. What's the use case?

Reply:
Tests